src/chain/mod.rs

```diff
@@ -0,0 +1,38 @@
+use color_eyre::eyre::{eyre, Result};
+use std::net::TcpListener;
+use std::process::{Child, Command};
+
+use crate::KIT_CACHE;
+
+pub fn start_chain(port: u16) -> Result<Child> {
+    let child = match TcpListener::bind(("127.0.0.1", port)) {
+        Ok(_) => {
+            let child = Command::new("anvil")
+                .arg("--port")
+                .arg(port.to_string())
+                .arg("--load-state")
+                .arg(format!("{}/kinostate.json", KIT_CACHE))
+                .stdout(std::process::Stdio::piped())
+                .spawn()?;
+            Ok(child)
+        }
+        Err(e) => Err(eyre!("Port {} is already in use: {}", port, e)),
+    };
+
+    // TODO: read stdout to know when anvil is ready instead.
+    std::thread::sleep(std::time::Duration::from_millis(100));
+    child
+}
+
+/// kit chain, alias to anvil
+pub async fn execute(port: u16) -> Result<()> {
+    Command::new("anvil")
+        .arg("--port")
+        .arg(port.to_string())
+        .stdout(std::process::Stdio::piped())
+        // .arg("--load-state")
+        // .arg("./kinostate.json")
+        .spawn()?;
+
+    Ok(())
+}
```

Comment:
We either need to
1. Move `start_chain()` into `build.rs`
2. Make `execute()` real: expose via commandline, etc.

As of right now I'm not convinced that there is a big win to wrapping a single command in `kit`. If there were more functionality, sure. Is there something I'm missing that would make this very desirable to have as its own `kit` command?

Reply:
I think number 2 would be preferable, as currently if kit f boots anvil in the background, I haven't figured out a way to have a command suddenly show the output in the fake terminal. So instead of running anvil --load-state in a specific directory, I thought kit c (kit chain) could be used, wrapping 3 implicit commands:
1. check for kinostate.json in kit_cache dir
2. if not present, write it there from kit binary
3. run anvil from there